jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -229,11 +168,3 @@ def test_spectrum1d_parse(spectrum1d, cubeviz_helper):
 def test_numpy_cube(cubeviz_helper):
     with pytest.raises(NotImplementedError, match='Unsupported data format'):
         cubeviz_helper.load_data(np.ones(27).reshape((3, 3, 3)))
-
-
-def test_multiple_load(image_hdu_obj, cubeviz_helper):
-    cubeviz_helper.load_data(image_hdu_obj)
-
-    # first load should be succesful; subsequent attempts should fail
-    with pytest.raises(RuntimeError, match=r".?only one (data)?cube.?"):
```

Comment:
This test is merged with another test above. Loading is expensive, so it saves time to piggyback this on some other test with the same data already loaded.